bundle/manifests/oadp-operator.clusterserviceversion.yaml

```
@@ -178,7 +178,7 @@ metadata:
     operatorframework.io/arch.arm64: supported
     operatorframework.io/arch.ppc64le: supported
     operatorframework.io/arch.s390x: supported
-  name: oadp-operator.v99.0.0
+  name: oadp-operator.v1.2.0
```

@shubham-pampattiwar @kaovilai  , I'd argue this should be v1.2.2 since that's what we're currently building from this branch downstream -- and reflects the same workflow/usage of our other products (such as MTC, for example)

```suggestion
  name: oadp-operator.v1.2.2
```